File: Leetcode/algorithms/easy/2264_largest_3_same_digit_number_in_string/2264_largest_3_same_digit_number_in_string.cs
public class Solution {
    public string LargestGoodInteger(string num) {
        string maxGood = "";
        
        for (int i = 0; i < num.Length - 2; i++) {
            if (num[i] == num[i + 1] && num[i] == num[i + 2]) {
                string substr = num.Substring(i, 3);
                if (substr.CompareTo(maxGood) > 0) {
                    maxGood = substr;
                }
            }
        }
        
        return maxGood;
    }
}

File: Leetcode_past/algorithms/easy/2264_largest_3_same_digit_number_in_string/2264_largest_3_same_digit_number_in_string.ts
function largestGoodInteger(num: string): string {
    let maxGood: string = "";

    for (let i = 0; i < num.length - 2; i++) {
        if (num[i] === num[i + 1] && num[i] === num[i + 2]) {
            const substr: string = num.substring(i, i + 3);
            if (substr > maxGood) {
                maxGood = substr;
            }
        }
    }

    return maxGood;
}
